development/metamaskbot-build-announce.js

```diff
@@ -65,50 +76,62 @@ async function start() {
   // build the github comment content
 
   // links to extension builds
-  const platforms = ['chrome', 'firefox'];
-  const buildLinks = platforms
-    .map((platform) => {
-      const url =
-        platform === 'firefox'
-          ? `${BUILD_LINK_BASE}/builds-mv2/metamask-${platform}-${VERSION}.zip`
-          : `${BUILD_LINK_BASE}/builds/metamask-${platform}-${VERSION}.zip`;
-      return `<a href="${url}">${platform}</a>`;
-    })
-    .join(', ');
-  const betaBuildLinks = `<a href="${BUILD_LINK_BASE}/builds-beta/metamask-beta-chrome-${VERSION}.zip">chrome</a>`;
-  const flaskBuildLinks = platforms
-    .map((platform) => {
-      const url =
-        platform === 'firefox'
-          ? `${BUILD_LINK_BASE}/builds-flask-mv2/metamask-flask-${platform}-${VERSION}-flask.0.zip`
-          : `${BUILD_LINK_BASE}/builds-flask/metamask-flask-${platform}-${VERSION}-flask.0.zip`;
-      return `<a href="${url}">${platform}</a>`;
-    })
-    .join(', ');
-  const mmiBuildLinks = platforms
-    .map((platform) => {
-      const url = `${BUILD_LINK_BASE}/builds-mmi/metamask-mmi-${platform}-${VERSION}-mmi.0.zip`;
-      return `<a href="${url}">${platform}</a>`;
-    })
-    .join(', ');
-  const testBuildLinks = platforms
-    .map((platform) => {
-      const url =
-        platform === 'firefox'
-          ? `${BUILD_LINK_BASE}/builds-test-mv2/metamask-${platform}-${VERSION}.zip`
-          : `${BUILD_LINK_BASE}/builds-test/metamask-${platform}-${VERSION}.zip`;
-      return `<a href="${url}">${platform}</a>`;
-    })
-    .join(', ');
-  const testFlaskBuildLinks = platforms
-    .map((platform) => {
-      const url =
-        platform === 'firefox'
-          ? `${BUILD_LINK_BASE}/builds-test-flask-mv2/metamask-flask-${platform}-${VERSION}-flask.0.zip`
-          : `${BUILD_LINK_BASE}/builds-test-flask/metamask-flask-${platform}-${VERSION}-flask.0.zip`;
-      return `<a href="${url}">${platform}</a>`;
-    })
-    .join(', ');
+  const buildMap = {
```

Comment:
Specifying the labels and builds declaratively made it easier to apply the same validation and formatting to all of them.